src/Microsoft.Health.Dicom.Api/Features/Filters/BodyModelStateValidatorAttribute.cs

```diff
@@ -18,7 +18,11 @@ public override void OnActionExecuting(ActionExecutingContext context)
             EnsureArg.IsNotNull(context, nameof(context));
             if (!context.ModelState.IsValid)
             {
-                throw new InvalidRequestBodyException(Environment.NewLine + string.Join(Environment.NewLine, context.ModelState.SelectMany(x => x.Value.Errors).Select(x => x.ErrorMessage)));
+                var message = string.Join(Environment.NewLine,
+                    context.ModelState.Where(x => x.Value.ValidationState == AspNetCore.Mvc.ModelBinding.ModelValidationState.Invalid)
+                     .Select(x => $"\t{x.Key} - {x.Value.Errors.First().ErrorMessage}"));
+
+                throw new InvalidRequestBodyException(Environment.NewLine + message);
```

Comment:
This was the existing behavior, but it's kind of weird that that a newline is passed as the first element. If this is internally concatenating strings, perhaps the `InvalidRequestBodyException` should take care of inserting the newline.

Reply:
I found first error message should be good enough. will simplify